onprc_ehr/resources/schemas/onprc_ehr.xml

```diff
@@ -1335,4 +1335,33 @@
         </columns>
     </table>
 
+
+    <table tableName="BSUageclass" tableDbType="TABLE" useColumnOrder="true">
+        <tableTitle>BSU Age Class</tableTitle>
+        <javaCustomizer class="org.labkey.ehr.table.DefaultEHRCustomizer" />
+        <auditLogging>DETAILED</auditLogging>
+        <columns>
```

Comment:
The automated tests are failing because the XML list of columns is incomplete:

Table "BSUageclass", column "rowid" missing from XML